vm/src/frame.rs

```diff
@@ -203,6 +203,7 @@ enum BlockType {
         end: bytecode::Label,
         context_manager: PyObjectRef,
     },
+    ExceptHandler {},
```

Comment:
Does CPython has a concept like this as well? Did you take the idea from there?

Reply:
No. I just wanted something that will allow me to know I finished the except handler.